go.mod

```diff
@@ -59,7 +59,7 @@ require (
 	github.com/evanphx/json-patch/v5 v5.9.11 // indirect
 	github.com/exponent-io/jsonpath v0.0.0-20210407135951-1de76d718b3f // indirect
 	github.com/felixge/httpsnoop v1.0.4 // indirect
-	github.com/fsnotify/fsnotify v1.7.0 // indirect
+	github.com/fsnotify/fsnotify v1.8.0 // indirect
```

Comment:
I don't see controller-runtime bumped in this commit.

Reply:
😣    Not sure how it ended up in https://github.com/openshift/windows-machine-config-operator/pull/2691/commits/4c95399cc53a85730f3e640e77dc312a1623a21c

will wait for the current test run to finish and update.

Reply:
Addressed, ended up squashing the commit for better handling.

Reply:
@mansikulkarni96  PTAL